module/items/skill/data.js

```diff
@@ -116,8 +125,7 @@ export class CoC7Skill extends CoC7Item {
    * This is the skill's value after active effects have been applied
    */
   get value () {
-    const value = this.parent?.system.skills?.[`${this.system.skillName}`]
-      ?.value
+    const value = this.parent?.system.skills?.[`${this.itemIdentifier}`]?.value
```

Comment:
This error was affecting not only the display on the sheet, but also dice rolls.